src/ChainSafe.GamingSDK.EVM/RPC/Providers/Web3Provider.cs

```diff
@@ -6,27 +6,27 @@
 
 namespace Web3Unity.Scripts.Library.Ethers.Providers
 {
-    public class Web3Provider : JsonRpcProvider
-    {
-        private readonly IExternalProvider _provider;
-
-        public Web3Provider(IExternalProvider provider,
-            Network.Network network = null)
-        {
-            if (provider == null)
-            {
-                throw new Exception($"missing provider {nameof(provider)}");
-            }
-
-            // TODO: why make an unused variable?
-            var path = provider.GetPath();
-
-            _provider = provider;
-        }
-
-        public override async Task<T> Send<T>(string method, object[] parameters = null)
-        {
-            return await _provider.Send<T>(method, parameters);
-        }
-    }
+    // public class Web3Provider : JsonRpcProvider
```

Comment:
Web3Provider was implemented to replace the current Web3GL Template implementation currently being used. I refrained from using it in v2 because it would have been to many changes to the current workflow. So, if you wish you can remove it. There shouldn't be any adverse effects on it's removal.

Reply:
@oleksandrchainsafe sounds like you can remove this.

Reply:
Removed together with WebGLProvider (it was commented out even before).
There is also IExternalProvider left, that is not being used anywhere. What do you say? Should we remove it also?

Reply:
Let's leave that for later and get this merged in ASAP.